tools/development/helpers/docs.sh

```
@@ -20,7 +20,11 @@ if [[ $1 == "--help" ]]; then
     exit 0
 fi
 
-python${OSTK_PYTHON_VERSION} -m pip install -r requirements.txt
+# Find and use the venv that was created when building the package.
+package_directory=(${project_directory}/build/bindings/python/OpenSpaceToolkit*Py-python-package-${OSTK_PYTHON_VERSION})
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)